.env

```diff
@@ -1,10 +1,10 @@
 REACT_APP_CHAIN_1=Rialto
 REACT_APP_CHAIN_2=Millau
-REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1=wss://wss.rialto.brucke.link
-REACT_APP_SUBSTRATE_PROVIDER_CHAIN_2=wss://wss.millau.brucke.link
+REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1=ws://127.0.0.1:9944
```

Comment:
This file shouldn't be changed.

Please use `.env.local` to have your own links.

Reply:
That was a draft PR - :)